assets/component-predictive-search.css

```diff
@@ -1,4 +1,5 @@
 .predictive-search {
+  width: calc(100% + 0.2rem);
```

Comment:
This only needs to be applied on desktop for the header search. And on mobile and desktop for the search in the search page/template. So I think we would need to revert to what it was before.